src/components/markdown/CodeBlock/CodeBlock.tsx

```diff
@@ -102,6 +102,11 @@ class CodeBlock extends React.Component<
 
   constructor(props) {
     super(props);
+
+    if (typeof props.isCodeBlockExpandable == 'undefined') {
+      props.isCodeBlockExpandable = true;
+    }
+
```

Comment:
It's bad practice to modify a component's props inside itself. Create a variable instead (you can also use the nullish coalescing operator).
```suggestion
    const isCodeBlockExpandable = props.isCodeBlockExpandable ?? true;

```